frontend/src/app/components/routes/main-view/components/meetings/components/current-meeting-content/current-meeting-content.component.scss

```diff
@@ -3,6 +3,33 @@
 :host {
     .date {
         padding: $small-spacing;
-        font-size: $very-large-font;
+        font-size: $normal-font;
+    }
+}
+
+.container {
```

Comment:
Nie lepiej po prostu walnąć klasę `flex-center`?
Podobnie w kilku innych przypadkach, trochę duplikacji się oszczędzi.

Reply:
Lepiej, zapominam ciągle, że ją mamy xd

Reply:
row to domyślna ale w kilku miejscach nie łapało tego no i nie chciałem już na to tracić czasu i po prostu napisałem
ale to prawdopodobnie sytuacje gdzie w jednym miejscu gdzies byl kolumn i trzeba bylo dac row'a, więc to też do zmiany ale nie wiem czy w tym tasku bo są inne rzeczy do roboty chyba

Reply:
Może coś docker nie domógł
Bo pamiętam, że nie działało
Teraz sprawdziłem i bez row'a jest gites